src/main/java/net/pms/image/ImagesUtil.java

```diff
@@ -1333,7 +1333,7 @@ protected static Image transcodeImage(
 			throw new UnknownFormatException("Unable to read image format", e);
 		}
 
-		if (inputResult.bufferedImage == null || inputResult.imageFormat == null) { // ImageIO doesn't support the image format
+		if (inputResult.bufferedImage == null) { // ImageIO doesn't support the image format
```

Comment:
Why don't we need to check for that?

Reply:
@SubJunk this was only for the testing. I returned it back in the final version